### ISSUE TITLE
[ENH]: add one or more isoluminant colormap(s)

### ISSUE THREAD

**Issue by boffi** [matplotlib/matplotlib#24909]
### Problem

Is it possible to add one or more [isoluminant colormap(s)](https://colorcet.holoviz.org/user_guide/Continuous.html#:~:text=swatches(group%3D%27isoluminant%27)) to the colormaps already present in Matplotlib?

Of course, I could install [colorcet](https://colorcet.holoviz.org/) and use its isoluminant colormaps, but Matplotlib has colormaps of many sorts, aimed at all the different types of usage, except isoluminant colormaps.

Thank you for everything you've done ፨ gb

**Comment by oscargus** [comment]
I mark this as a good first issue, but medium since one require a bit of both how to specify colormaps and how to generate isoluminant colormaps.

**Comment by oscargus** [comment]
Yes, there are indeed several things to consider before adding one as it is "quite permanent" once in. However, the general concept of having one isoluminant colormap (only remains to decide which), seems like a good idea to me as there isn't one. (Which for sure is not the same thing as everyone else agrees with it.)

**Comment by tacaswell** [comment]
I removed good first issue.  The technical work is minimal, the consensus building is not.

**Comment by jklymak** [comment]
@boffi do you have references to isoluminant maps, know of a community that uses them, and if they have standards for what appropriate defaults would be?

**Comment by boffi** [comment]
My only reference to isoluminant colomaps (i.c.m.) is the documentation of
Colorcet, that I've linked in the original post.
With respect to which communities uses the i.c.m., my source is again
Colorcet's documentation, that explains that i.c.m. are in use in shaded
maps, as the artificially shaded maps used in geographical cartography,
where a change in luminance across the colormap could lead to a
misinterpretation of the relief.
Good night from Europe % gb

**Comment by ianthomas23** [comment]
It is quite simple to use `colorcet` colormaps in `matplotlib`, here is a simple example using one of the isoluminant colormaps:

```python
import colorcet
import matplotlib.pyplot as plt

fig, ax = plt.subplots()
cs = ax.contourf([[0, 0.5],[0.5, 1]], cmap=colorcet.cm.isolum)
fig.colorbar(cs, ax=ax)
plt.show()
```
<img width="389" alt="Screenshot 2023-01-27 at 09 02 57" src="https://user-images.githubusercontent.com/580326/215048184-bdef6a91-b958-4331-a91c-6a08fd8b1804.png">

Disclaimer: I am a `colorcet` maintainer.
